ksql-engine/src/main/java/io/confluent/ksql/function/udtf/array/Explode.java

```diff
@@ -55,8 +60,21 @@
     return explode(input);
   }
 
+  @Udtf(schemaProvider = "provideSchema")
+  public List<BigDecimal> explodeBigDecimal(final List<BigDecimal> input) {
+    return explode(input);
+  }
+
   private <T> List<T> explode(final List<T> list) {
```

Comment:
I'm wondering if we could leverage UDF generics here (i.e. see `AsArray`) - probably out of scope for this PR, but we should make sure (or document otherwise) whether or not UDTFs support the inferred generics

Reply:
I initially tried to code it with generics and couldn't get it to work, but I just tried again, and it works! ;) So not sure what I was doing wrong first time.

Reply:
done